README.md

```diff
@@ -9,6 +9,52 @@ images as well as for information regarding contributing and issues.
 The [Docker Hub page] is generated/maintained over in the
 [docker-library/docs repository], specifically in [the `dart` directory].
 
+## Using this image
+
+We recommend creating small runtime images by leveraging Dart's support for
+ahead-of-time (AOT) [compilation to executables][]. This enables creating small
+runtime images (~10 MB).
+
+The following `Dockerfile` performs two steps:
+
+1. Using the Dart SDK in the `dart:stable` image, compiles your server
+(`bin/server.dart`) to an executable (`server`).
+
+1. Assembles the runtime image by combining the compiled server with the Dart VM
+runtime and it's needed dependencies located in `/runtime/`.
+
+```
+# Use latest stable channel SDK.
+FROM dart:stable AS build
+
+# Resolve app dependencies.
+WORKDIR /app
+COPY pubspec.yaml .
```

Comment:
Should be `COPY pubspec.* .` for apps (shown [here](https://github.com/dart-lang/dart-docker/blame/794ca592448d784ef40ad773f3f2b6217720a2b7/README.md#L17)).

Reply:
Done